CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 DATE: _unreleased_
 
+* \#188: Upgraded to JTS 1.17.0.  This JTS release has a small [API change](https://github.com/locationtech/jts/blob/master/doc/JTS_Version_History.md#api-changes).  
+  LocationTech JTS has updated its license and is now dual-licensed as EPL 2.0 and EDL 1.0 (a BSD style license).
```

Comment:
JTS's license has not changed for 1.17 so why mention it here?

Reply:
I thought it had changed....  I've removed the note (it sounds like you didn't see the change in an obvious place).